po/es.po

```diff
@@ -5139,10 +5139,8 @@ msgstr "Borra el desglose efectivo"
 
 #. Translators: This is the label of a toolbar button. So keep it short.
 #: gnucash/gnome/gnc-plugin-page-register.c:382
-#, fuzzy
-#| msgid "Split"
 msgid "Split View"
-msgstr "Desglosar"
+msgstr "Dividir la vista"
```

Comment:
@gallegonovato Thanks for informing us of an ambiguity. Sorry you have choosen the wrong meaning, Not the view is divided, but the transaction details (splits) are shown.

@CWehli Perhaps <q>{Show|Display} splits</q> would be better?
FYI [Bug 799031—Ledger types in View menu should be renamed](https://bugs.gnucash.org/show_bug.cgi?id=799031) is in the same context.

Reply:
Good afternoon @fellen

But which chain is it? 

I say this in case you can look for another translation. But if it is an ambiguity it can cause problems, when translated?

Best regards

Reply:
Are you asking for [its location in Weblate](https://hosted.weblate.org/translate/gnucash/gnucash/es/?checksum=f198e8452594ea9c)?

Yes, CWehli and I  tried to resolve another ambiguity an d created a new one. So i started the discussion with him.

Reply:
> Perhaps "{Show|Display} splits" would be better?

`Show Split` is a good choice in my opinion. Tthe main thing is that it prevents misunderstandings.
But then we should also consider whether to adjust the German translation.